Rakefile

```diff
@@ -99,6 +99,11 @@ task :clean do
   end
 end
 
+desc "Build #{Bundler::GemHelper.gemspec.full_name} and move it to local ruby/ruby project's bundled gems folder"
+task build_for_local_ruby: :build do
```

Comment:
Can we use better task name something like `local_ruby:build`?

Reply:
Renamed it to `build:local_ruby`